content/en/workflows/trigger.md

```diff
@@ -14,18 +18,22 @@ further_reading:
   text: "Find out more about Monitors"
 ---
 
-You can trigger a workflow manually, on a schedule, or using a variety of automated methods such as a Datadog Monitor, or a Security Signal Notification Rule. Before you can trigger a workflow using automation, you must associate the workflow with a unique service account. For more information on setting up a service account for your workflows, see [Set up workflows][1].
+{{< beta-callout url="https://forms.gle/VEjerYVQ2QJhauZ57" >}}
+  Workflows are in public beta. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.
+{{< /beta-callout >}}
+
+You can trigger a workflow manually, on a schedule, or using a variety of automated methods such as a Datadog Monitor, or a Security Signal Notification Rule. Before you can trigger a workflow on a schedule, or using automation, you must associate the workflow with a unique service account. For more information on Workflows service accounts, see [Service accounts for Workflows][1].
```

Comment:
In my mental model, a schedule is one of the automated methods. (And I think this is also consistent with how the UI frames it, since you click "Add an automated trigger" in order to add a schedule.) So I might say something like:

> You can trigger a workflow manually or using a variety of automated methods such as a recurring schedule, a Datadog Monitor, or a Security Signal Notification Rule. Before you can trigger a workflow using one of these automated methods, ...

But if you've gotten different guidance from others, feel free to leave as-is!